diff -uNr apk-tools-2.10.3/src/clandro_prefix.h apk-tools-2.10.3.mod/src/clandro_prefix.h
--- apk-tools-2.10.3/src/clandro_prefix.h	1970-01-01 03:00:00.000000000 +0300
+++ apk-tools-2.10.3.mod/src/clandro_prefix.h	2018-11-19 17:42:52.188760273 +0200
@@ -0,0 +1,6 @@
+#ifndef __TERMUX_PREFIX_H
+#define __TERMUX_PREFIX_H
+
+#define CLANDRO_PREFIX_RELATIVE "data/data/com.zk.clandro/files/usr/"
+
+#endif
